substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/phases/IntrinsifyMethodHandlesInvocationPlugin.java

```diff
@@ -504,6 +512,12 @@ private void processInvokeWithMethodHandle(GraphBuilderContext b, Replacements r
         }
     }
 
+    public List<Pair<ResolvedJavaMethod, Integer>> getContext(GraphBuilderContext b) {
```

Comment:
This is not really the whole context. It needs a better name.

Reply:
Maybe getSingletonContext?